cmd/activator/main.go

```diff
@@ -183,8 +183,7 @@ func main() {
 
 		tlsConf := &tls.Config{
 			RootCAs:            pool,
-			InsecureSkipVerify: false,
-			ServerName:         certificates.FakeDnsName,
+			InsecureSkipVerify: true,
```

Comment:
My different attempts to perform `// #nosec G402` or `// #nosec` failed to remove the security warning - any ideas??